helm/oauth2-proxy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 7.8.3
+version: 7.9.3
```

Comment:
Please `7.9.0` 🙏🏻

Reply:
fixed in [ca8385f](https://github.com/oauth2-proxy/manifests/pull/270/commits/ca8385f9dac9f7443d1b7176604bc7aec1c2bf72)